router-tests/query_plans_test.go

```diff
@@ -244,8 +245,8 @@ func TestQueryPlans(t *testing.T) {
 					}`,
 			})
 
-			require.Contains(t, res.Body, "query Requires__mo_o_d__1")
-			require.Contains(t, res.Body, "query Requires__av_ai_la_bi_lit_y__2")
+			require.Contains(t, res.Body, "query Requires__mo_o_d")
+			require.Contains(t, res.Body, "query Requires__av_ai_la_bi_lit_y")
```

Comment:
What is the correct name of the operation in the end? Apollo had numbers. How do I define the nesting of a query now? It would be cool to be able to access the name of the operation in the module, not just `r`